types/pify/index.d.ts

```diff
@@ -8,7 +8,8 @@ interface PifyOptions {
     multiArgs?: boolean,
     include?: [string | RegExp],
     exclude?: [string | RegExp],
-    excludeMain?: boolean
+    excludeMain?: boolean,
+    errorFirst?: boolean
```

Comment:
Can you add [promiseModule](https://github.com/sindresorhus/pify#promisemodule) as well?

Reply:
And remove both line 15 and 16 because promiseModule can't be passed in anymore as second argument.